### ISSUE TITLE
Snackbar not working in Safari

### ISSUE THREAD

**Issue by jadebuckwalter** [Aspine/aspine#291]
Add support for the snackbar in Safari (right now the use of OOP and the "static" keyword is preventing the snackbar from loading).

**Comment by MohanMaker** [comment]
After a bit of research, I found that when I turn off my adblocker in Safari the problem goes away and the snack bar appears. Do you happen to have an adblocker as well @jadebuckwalter, and do you want to test if the problem goes away when you turn the adblocker off?

**Comment by tektaxi** [comment]
@MohanMaker I don't have an ad blocker on in Safari and It still doesn't work for me either. What version of Safari/Mac OS are you running? I'm on Big Sur.

**Comment by psvenk** [comment]
@tektaxi @MohanMaker It may be helpful to clarify with the exact version number from "About this Mac", in case any changes were made since the initial release of Big Sur. I find it strange that this works on Safari for @MohanMaker, because [caniuse.com](https://caniuse.com/mdn-javascript_classes_static) says that no stable version of Safari supports the `static` keyword. Are you on the Safari Technology Preview?

**Comment by tektaxi** [comment]
@psvenk @MohanMaker To clarify, I'm on 11.0.1

**Comment by jadebuckwalter** [comment]
@MohanMaker @tektaxi @psvenk I have an ad blocker, but it's through the network rather than Safari itself, so it shouldn't matter which browser I'm using. (I'm on Big Sur 11.2.3, by the way.) It seems more plausible that the `static` keyword would be the culprit here rather than an ad blocker.

**Comment by MohanMaker** [comment]
@jadebuckwalter @tektaxi @psvenk: Sorry, my mistake, it seems to be not working regardless of the adblocker. I agree that the static keywork is probably the issue.

**Comment by andOrlando** [comment]
I don't believe the static keyword is super important and whatever is static can probably just be extracted to a global variable or constant or something for the sake of compatibility
